File: b836(king_flower)/Python/king_flower.py
from sys import stdin
n, m = 0, 0
while True:
    try: n, m = map(int, stdin.readline().split())
    except: break
    if m == 0:
        print('Go Kevin!!')
    elif (n - 1) % m != 0:
        print('No Stop!!')
    else:
        times = ((m - 2) + ((m - 2) ** 2 + 8 * m * n) ** 0.5) / (2 * m)
        if int(times) == times: print('Go Kevin!!')
        else: print('No Stop!!')

File: b836(king_flower)/C++/king_flower.cpp
#include <iostream>
#include <cmath>

#define Go std::cout << "Go Kevin!!" << '\n'
#define No std::cout << "No Stop!!" << '\n'

int main(int argc, char const *argv[]) {
  long long n, m;
  while (std::cin >> n >> m) {
    if (m == 0) Go;
    else if ((n - 1) % m != 0) No;
    else {
      double times = ((m - 2) + sqrt((m - 2) * (m - 2) + 8 * m * n)) / (2 * m);
      if ((int)times == times) Go;
      else No;
    }
  }
  return 0;
}
